frontend/termbox/main.go

```diff
@@ -456,7 +456,7 @@ func (t *tbfe) renderthread() {
 	}
 
 	for range t.dorender {
-		log.Finest("Rendering")
+		// log.Finest("Rendering")
```

Comment:
It seemed extraneous and was cluttering up the log output. Happy to add it back in if we feel strongly about it.